package.json

```diff
@@ -24,21 +24,18 @@
     "index.js"
   ],
   "devDependencies": {
-    "babel": "^6.1.18",
-    "babel-preset-es2015": "^6.1.18",
-    "babel-tape-runner": "^2.0.0",
+    "ava": "^0.11.0",
```

Comment:
> as such I'd like to try and keep compatibility of all these NodeJS versions for both *nix and Windows.

Sgtm.

>  I've a suspicion this could actually be an AppVeyor issue rather than ours!

Agreed. Previously version 4 failed and 5 passed. I restarted the test a few minutes ago and 4 now passes, but 5 fails :(